composer.json

```diff
@@ -19,7 +19,7 @@
         "symfony/polyfill-php70": "^1.19"
     },
     "require-dev": {
-        "php-parallel-lint/php-parallel-lint": "1.0.0",
+        "php-parallel-lint/php-parallel-lint": "^1.0.0",
```

Comment:
Remove this change. If you want this change, submit in separate PR for discussion.

Reply:
Separated from this PR:
- https://github.com/zf1s/zf1/pull/58

Reply:
Removed